[2023-06-04T00:34:25.556+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-03T00:00:00+00:00/task_id=runme_0 permission to 509
[2023-06-04T00:34:25.558+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-03T00:00:00+00:00/task_id=runme_0 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-03T00:00:00+00:00/task_id=runme_0'
[2023-06-04T00:34:25.604+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-03T00:00:00+00:00/task_id=runme_0 permission to 509
[2023-06-04T00:34:25.605+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-03T00:00:00+00:00/task_id=runme_0 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-03T00:00:00+00:00/task_id=runme_0'
[2023-06-04T00:34:25.665+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.runme_0 scheduled__2023-06-03T00:00:00+00:00 [queued]>
[2023-06-04T00:34:25.681+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.runme_0 scheduled__2023-06-03T00:00:00+00:00 [queued]>
[2023-06-04T00:34:25.682+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-04T00:34:25.704+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): runme_0> on 2023-06-03 00:00:00+00:00
[2023-06-04T00:34:25.713+0000] {standard_task_runner.py:57} INFO - Started process 18093 to run task
[2023-06-04T00:34:25.716+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'runme_0', 'scheduled__2023-06-03T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpngji747f']
[2023-06-04T00:34:25.719+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask runme_0
[2023-06-04T00:34:25.802+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-03T00:00:00+00:00/task_id=runme_0 permission to 509
[2023-06-04T00:34:25.804+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-03T00:00:00+00:00/task_id=runme_0 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=example_bash_operator/run_id=scheduled__2023-06-03T00:00:00+00:00/task_id=runme_0'
[2023-06-04T00:34:25.808+0000] {task_command.py:410} INFO - Running <TaskInstance: example_bash_operator.runme_0 scheduled__2023-06-03T00:00:00+00:00 [running]> on host cd81537d5f1e
[2023-06-04T00:34:25.946+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_0' AIRFLOW_CTX_EXECUTION_DATE='2023-06-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-03T00:00:00+00:00'
[2023-06-04T00:34:25.949+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-04T00:34:25.950+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "example_bash_operator__runme_0__20230603" && sleep 1']
[2023-06-04T00:34:25.964+0000] {subprocess.py:86} INFO - Output:
[2023-06-04T00:34:25.969+0000] {subprocess.py:93} INFO - example_bash_operator__runme_0__20230603
[2023-06-04T00:34:26.972+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-04T00:34:27.008+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_0, execution_date=20230603T000000, start_date=20230604T003425, end_date=20230604T003427
[2023-06-04T00:34:27.055+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-04T00:34:27.127+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
